sklearn/datasets/tests/test_svmlight_format.py

```diff
@@ -553,3 +553,22 @@ def test_load_offset_exhaustive_splits():
 def test_load_with_offsets_error():
     with pytest.raises(ValueError, match="n_features is required"):
         _load_svmlight_local_test_file(datafile, offset=3, length=3)
+
+
+def test_multilabel_y_explicit_zeros(tmp_path):
```

Comment:
Can we include a comment here stating what this is testing? I think it was the that explicit zeros in `y` is ignored?